dali/operators/decoder/nvjpeg/decoupled_api/nvjpeg_decoder_decoupled_api.h

```diff
@@ -35,6 +35,84 @@
 
 namespace dali {
 
+namespace memory {
+
+using Buffer = std::pair<void*, size_t>;
```

Comment:
Maybe it is better to create a struct with meaningful fields?

Reply:
done